src/main/java/com/cburch/logisim/prefs/AppPreferences.java

```diff
@@ -574,6 +576,8 @@ public static void setDefaultGridColors() {
       create(new PrefMonitorInt("componentSecondaryColor", DEFAULT_COMPONENT_SECONDARY_COLOR));
   public static final PrefMonitor<Integer> COMPONENT_GHOST_COLOR =
       create(new PrefMonitorInt("componentGhostColor", DEFAULT_COMPONENT_GHOST_COLOR));
+  public static final PrefMonitor<Integer> COMPONENT_ICON_COLOR =
+          create(new PrefMonitorInt("componentIconColor", DEFAULT_COMPONENT_ICON_COLOR));
```

Comment:
A wrapped line should be indented 4 characters. That would also make it match the line above.